testsuite/gnat2goto/tests/string_assert/test.out

```diff
@@ -0,0 +1 @@
+ALL XFAIL CBMC fails types of expressions on each side of equality should match
```

Comment:
I think this should be in test.opt, not test.out ?

Reply:
https://github.com/diffblue/gnat2goto/pull/93#discussion_r242583565 .. now I'm confused.

Reply:
I just edited my comment about to hopefully clarify - expected passing output goes in 'test.out' and any 'options' about the testcase, such as XFAIL, goes in 'test.opt' See https://github.com/diffblue/gnat2goto/tree/master/testsuite/gnat2goto/tests/recursion for an example.

Reply:
@chrisr-diffblue Ok, I think I understand it now.